### ISSUE TITLE
Implementation of WorldExpo'10 dataset

### ISSUE THREAD

**Issue by desenzhou** [leeyeehoo/CSRNet-pytorch#75]
Hi, I'm using CSRNet for crowd counting on world Expo'10 dataset. However, I cannot reproduce the results of the paper(8.6 MAE), I only got a MAE around 10.2. Could you please provide the implementation of world Expo'10 dataset(e.g. density map generation or how to utilize the ROI and evalutation), or at least provide some implementation details( learning rate, batch size ect.). Thank you so much!!

**Comment by desenzhou** [comment]
In addition, as world Expo'10 contains 5 scenes. During evaluation, do you use the same model for 5 scenes, or one model for one scene and 5 models in total. Thanks!

**Comment by 845601575** [comment]
Hi, I want to use my dataset to train the model,   
but I don't know what the 'python train.py train.json val.json 0 0' means
could you explain it to me?
thank you very much!
